package.json

```diff
@@ -21,19 +21,25 @@
     "@fortawesome/free-solid-svg-icons": "^6.5.1",
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@radix-ui/react-dropdown-menu": "^2.0.6",
+    "@radix-ui/react-slot": "^1.0.2",
     "axios": "^1.6.7",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
     "cryptr": "^6.3.0",
     "date-fns": "^3.3.1",
+    "embla-carousel-react": "^8.0.0",
     "iso-639-3": "^3.0.1",
     "jwt-decode": "^4.0.0",
+    "lucide-react": "^0.343.0",
     "next": "14.1.0",
     "next-auth": "^4.24.5",
     "react": "^18",
     "react-dom": "^18",
+    "react-icons": "^5.0.1",
+    "sharp": "^0.33.2",
```

Comment:
can you double check where does this `sharp` packages is being used? I have not seen it being used in this PR

Reply:
These components, the button.tsx, carouse.tsx are from shadcn.
In this PR I won't use it as I will need something from Younes's pr to be merged before. Will delete it.
(Btw this explains some of the comments you did below regarding the colours, because these are shadcn components)